node/sc/bridge_test.go

```diff
@@ -632,7 +632,7 @@ func TestExtendedBridgeAndCallbackERC721(t *testing.T) {
 		assert.Equal(t, bobAcc.From, ev.To)
 
 		// HandleERC20Transfer
-		tx, err = b.HandleERC721Transfer(bridgeAccount, ev.Raw.TxHash, ev.From, ev.To, ev.TokenAddress, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.Uri, ev.ExtraData)
+		tx, err = b.HandleERC721Transfer(bridgeAccount, ev.Raw.TxHash, ev.From, ev.To, ev.TokenAddress, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, "", ev.ExtraData)
```

Comment:
Minor comment:
We can test `TokenURI()` method here.

Reply:
Yes. But, I wanted to test with `bridgeInfo.processingPendingRequestEvents()`. :)

Reply:
So I added additional TC. :)